lib/src/sdk.ts

```diff
@@ -330,11 +330,15 @@ export class ExchangeSDK {
       });
 
     if (getSellPayload) {
-      await decodeSellPayloadAndPost(
-        binaryPayload as string,
+      const newQuoteId = await decodeSellPayloadAndPost(
+        binaryPayload.toString(),
```

Comment:
Should we change `decodeSellPayloadAndPost` expected type and do the hole conversion in that function?

```suggestion
        binaryPayload,
```